File: co.edu.uniandes.fixml.toCSharp/test/test6.xml/Qty.cs


// Class for tag Qty
class Qty {

	string _Long;
	string _Short;
	string _Typ;
	
	
	// default constructor
	
	public Qty ( ) {   
		this._Long = "35";
		this._Short = "0";
		this._Typ = "SOD";
		
		
	}
	
	public Qty ( string Long, string Short, string Typ ) {
		this._Long = Long;
		this._Short = Short;
		this._Typ = Typ;
		
	}
	
	
} 

File: co.edu.uniandes.fixml.toC++/test/text2.xml/Qty.cpp
#include <iostream> 

using namespace std;

class Qty {

	private:
  	string Typ_;
  	string Long_;
  	string Short_;
 	
 	public: 

  	// default constructor
	Qty () {
		Typ_ = "SOD";
		Long_ = "35";
		Short_ = "0";
  	
  	}
  
  	// by parameters
  	Qty (		string Typ_p, string Long_p, string Short_p){
	  
		Typ_ = Typ_p;
		Long_ = Long_p;
		Short_ = Short_p;
	  
  	}
};